### PR TITLE
feat: enable passing props down to label in ControlledInput [Homer 25]

### PR DESCRIPTION
**Pull request by Chaoste**
# Purpose of PR

For migrating ControlledInputField, we want to pass styles down to the label component. Thus, we add the prop `labelProps` in the same manner as `inputProps`.

[See related comment](https://github.com/contentful/forma-36/pull/954/files#r637008561)

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/components/inputs/src/controlled-input/ControlledInput.tsx [hunk 1]
```diff
@@ -128,12 +138,9 @@ const _ControlledInput = (
       />
       {type === 'radio' ? (
         // eslint-disable-next-line jsx-a11y/label-has-associated-control
-        <label
-          className={cx(styles.ghost, styles.ghostRadioButton)}
-          htmlFor={id}
-        />
+        <label {...labelProps} className={labelClassnames} htmlFor={id} />
```
Comment (denkristoffer) [inline packages/components/inputs/src/controlled-input/ControlledInput.tsx:141]: Shouldn't this use the Label component?

Comment (Chaoste) [reply packages/components/inputs/src/controlled-input/ControlledInput.tsx]: Good point, I didn't know about Label when I was migrating the component :)

Comment (Chaoste) [reply packages/components/inputs/src/controlled-input/ControlledInput.tsx]: I tried but
```
export type LabelInternalProps = {
  /**
   * Label value to show
   */
  children: React.ReactText;
  ...
}
```

It doesn't allow me to pass actual components. I could only do it for the first of both cases which doesn't have children at all. But I need both to do the same so, `Label` doesn't work here :/

Comment (denkristoffer) [reply packages/components/inputs/src/controlled-input/ControlledInput.tsx:140]: Ah we should just change the type then, that was just me trying to narrow it down. Of course it should allow for icons.

**Review by denkristoffer — APPROVED** [review]

**Pull request merged**